From bd9377a5be5f632cd091c6bc4b0246d7556c8244 Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Wed, 23 Oct 2019 14:18:50 +0800
Subject: [PATCH] phy: add GVI/HDMI modes to the phy_mode enum

This patch adds more generic PHY modes to the phy_mode enum, to
allow configuring generic PHYs to the GVI/HDMI mode by
using the set_mode callback.

Change-Id: Ib859024d7bd59cd39bff5e06f2b2ca133b023414
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 include/linux/phy/phy.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index 5f7ed6d86e43..54c574cc5217 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -45,6 +45,8 @@ enum phy_mode {
 	PHY_MODE_VIDEO_TTL,
 	PHY_MODE_PCIE_RC,
 	PHY_MODE_PCIE_EP,
+	PHY_MODE_GVI,
+	PHY_MODE_HDMI,
 };
 
 /**
-- 
2.35.3

